src/constructs/ec2/security-groups.ts

```diff
@@ -1,42 +1,60 @@
 import type { CfnSecurityGroup, IPeer, SecurityGroupProps } from "@aws-cdk/aws-ec2";
 import { Peer, Port, SecurityGroup } from "@aws-cdk/aws-ec2";
-import { transformToCidrIngress } from "../../utils/security-groups";
 import type { GuStack } from "../core";
 
-export interface CidrIngress {
+/**
+ * A way to describe an ingress or egress rule for a security group.
+ *
+ * See [[transformToSecurityGroupAccessRule]] for a handy helper function.
```

Comment:
Maybe we should remind users to make use of `GuWazuhAccess`, `GuPublicInternetAccessSecurityGroup`, `GuHttpsEgressSecurityGroup` (if possible), in case people stumble across these docs first and start unnecessarily reimplementing common cases?